src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlCommandTest.cs

```diff
@@ -184,7 +184,7 @@ public void Constructor4()
             Assert.Null(cmd.Container);
             Assert.True(cmd.DesignTimeVisible);
             Assert.Null(cmd.Notification);
-#if NETFX
+#if NETFX && !NETSTANDARDREFERNCE
```

Comment:
I guess line [169](https://github.com/dotnet/SqlClient/blob/4c63e9ecd1d2097047e6c2bb66f957fc16ff81e2/src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlCommandTest.cs#L169) should be updated too.